### ISSUE TITLE
Login and SignUp Page

### ISSUE THREAD

**Issue by Akshima-Ghai** [Akshima-Ghai/OneEducationalAppForAll#4]
Make a Login and Signup page for the app

**Comment by shivshankar9** [comment]
Akshima ,,
can I try this.....

**Comment by Akshima-Ghai** [comment]
Go ahead @shivshankar9 . All the best

**Comment by amartyanambiar** [comment]
> Akshima ,,
> can I try this.....

Hey,
Do you know to handle the back-end? The firebase part?

**Comment by shivshankar9** [comment]
> > Akshima ,,
> > can I try this.....
> 
> Hey,
> Do you know to handle the back-end? The firebase part?

No idea bro ,
I am learning this too in same time as  I said already to 
Akshima,
I 'm just trying ...

**Comment by shivshankar9** [comment]
> > Akshima ,,
> > can I try this.....
> 
> Hey,
> Do you know to handle the back-end? The firebase part?

If you have experience in that you can take this issue,
I have no problem...
As we all came here to learn....

**Comment by amartyanambiar** [comment]
Actually , I only know the front-end part. Back-end zero.

Soo we could we could work on the UI anyways.

**Comment by shivshankar9** [comment]
I don't think here mentioned anything about back-end
@Akshima-Ghai do I have to work back-end too🤔

**Comment by Akshima-Ghai** [comment]
Yes, guys we also have to work on the backend part too, it's very using firebase. You can learn it from can video tutorial from YouTube, just type login -signup using firebase, you will get enough resources to learn. If u still got any doubt, u can ask me
@shivshankar9 @amartyanambiar

**Comment by shivshankar9** [comment]
Okay akshima
We will try our best...

**Comment by amartyanambiar** [comment]
![image](https://user-images.githubusercontent.com/51471924/124384706-2ff85e00-dcf0-11eb-8ae5-24ba93f310c5.png)
![image](https://user-images.githubusercontent.com/51471924/124384738-51594a00-dcf0-11eb-93fc-a9e5bd445bfb.png)
This is the ui for login with the temporary logo.If it's ok. I'll send a PR. The ui for signup will be same.

**Comment by Akshima-Ghai** [comment]
improve the color theme , add some designs and make it more attractive @amartyanambiar . Check the link to get a better understanding
https://dribbble.com/shots/15727340-App-UI?utm_source=Clipboard_Shot&utm_campaign=D-studio&utm_content=App%20UI&utm_medium=Social_Share&utm_source=Clipboard_Shot&utm_campaign=D-studio&utm_content=App%20UI&utm_medium=Social_Share
https://dribbble.com/shots/15432259-Ocula-UI-UX-Login-Screen-Interaction-3?utm_source=Clipboard_Shot&utm_campaign=samoctav&utm_content=Ocula%20%E2%80%94%20UI%2FUX%20Login%20Screen%20Interaction%203&utm_medium=Social_Share&utm_source=Clipboard_Shot&utm_campaign=samoctav&utm_content=Ocula%20%E2%80%94%20UI%2FUX%20Login%20Screen%20Interaction%203&utm_medium=Social_Share

**Comment by amartyanambiar** [comment]
Okay, i get it. Should i make pr ..soo that others can work on it too?

**Comment by Akshima-Ghai** [comment]
@amartyanambiar ok , create a pr

**Comment by Priyanshu078** [comment]
Please assign this issue 
I will make the UI functional using the firebase backend for signup and login

**Comment by pratishthaa** [comment]
Is backend part added?

**Comment by Akshima-Ghai** [comment]
@Priyanshu078 issue assigned
